examples/jenkins/master-slave/README.md

```diff
@@ -65,7 +65,7 @@ Before you begin, ensure you have created the [default imagestreams](https://doc
    ready to be used.
 
 8. Now click on *Add to project* again and select the `jenkins-master` template.
-   Here you can pick your Jenkins administrator password, alternative Docker
+   Here you can pick your authentication plugin and if needed Jenkins administrator password, alternative Docker
```

Comment:
yeah ok i forgot about that.  at some point we should probably consider simplifying the master-slave example to just use the existing pre-built slaves instead of doing all the magic of building new slaves and such.